Clear-Your-Browser's-Local-Storage.md

```diff
@@ -6,9 +6,9 @@ In Chrome:
  * Mac OS: Cmd + Opt + J
 * Go to resources tab(chrome). 
  * There click on the localStorage link in the nav bar on the right.
-* Delete all the entries on the right side. 
+* Delete all the entries on the right side, or run this command in your browser's console to clear all entries from your localStorage: localStorage.clear();

```

Comment:
Just nitpicking here: \`localStorage.clear();\` so that it appears `localStorage.clear();` when rendered.